content/design/pages/landing-page.md

```diff
@@ -0,0 +1,215 @@
+
+Magna fermentum iaculis eu non. Faucibus in ornare quam viverra orci sagittis eu volutpat. At erat pellentesque adipiscing commodo elit. Vel elit scelerisque mauris pellentesque pulvinar pellentesque habitant. Turpis egestas integer eget aliquet nibh praesent tristique. Semper quis lectus nulla at volutpat diam ut.
+
+Turpis nunc eget lorem dolor sed viverra ipsum. Lobortis feugiat vivamus at augue eget arcu dictum varius duis. Quisque sagittis purus sit amet volutpat consequat mauris. Viverra nibh cras pulvinar mattis nunc sed blandit libero volutpat. Quis enim lobortis scelerisque fermentum dui faucibus in. Fames ac turpis egestas sed tempus urna et pharetra pharetra. Nisl nisi scelerisque eu ultrices vitae. Lectus vestibulum mattis ullamcorper velit sed ullamcorper. Felis bibendum ut tristique et. Ac turpis egestas sed tempus urna et pharetra. Lectus urna duis convallis convallis tellus. A cras semper auctor neque vitae. [Ullamcorper velit](//spotlightpa.org) sed ullamcorper morbi tincidunt ornare. Libero volutpat sed cras ornare arcu dui.
+
+
+*— Accepts **markdown***
+{{</supplemental>}}"""
+
+credit = """
+{{<featured/credit-long
+hed="Custom Credit-Long Shortcode"
+name="Wyatt Massey"
+role="Capitol Reporter"
+email="wmassey@spotlightpa.org"
+bio="Wyatt Massey investigates how Penn State University operates, including its influence in the region and state. He previously covered faith and religion for the Chattanooga Times Free Press in Southeast Tennessee."
+shortcode="{{<supplemental>}}"
+>}}"""
+
+form ="""
+{{<featured/promo-tips-form
+  hed="Send A Tip"
+  dek="Help support the Penn State Transparency Tracker by submitting tips or questions to reporter Wyatt Massey — via the submission form, email at [wmassey@spotlightpa.org](mailto:wmassey@spotlightpa.org), or one of the methods described below."
+>}}
+"""
+newsletter = """
+{{<newsletter/pennstatealert-full-form>}}
+"""
+[[blocks]]
+title = "Text Block Two"
+layout = "text-2"
+copy="""
+  {{<featured/large>}}Spotlight PA is a nonpartisan, independent newsroom paid for by grants from foundations and donations from individuals committed to robust investigative reporting. Spotlight PA editors and reporters operate independently of our funders and maintain editorial control over all of the content they produce. Funders do not have input into the selection of which stories Spotlight PA pursues or the reporting process for those stories. Funders do not review any content before publication or have any special access to reporters or newsroom leadership. In short, we answer to you, the public.{{</featured/large>}}
+
+  {{<featured/large>}}Funds for Spotlight PA are donated to the nonprofit <a href="https://www.lenfestinstitute.org/" class="text-yellow">Lenfest Institute for Journalism</a>, which then directs them to our work. As part of our gift acceptance policy, and consistent with our promise to be fully transparent, we do not accept anonymous donations.{{</featured/large>}}
```

Comment:
I don't like having the large shortcode here. The text block template should have whatever text styles it needs to look good without a shortcode. Also I think the underline of the hyperlink should be tweaked a bit. Make it fatter and move it down some. It probably needs some a new CSS class on the parent template so it can apply to any link in the block.